proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md

```diff
@@ -118,10 +118,32 @@ While `diagonalScreenSize` is mandatory, `aspectRatio` and `resolution` are opti
 In `SDLStreamingMediaConfiguration`:
 
 ```objectivec
-@property (strong, nonatomic, nullable) VideoStreamingRange *supportedLandscapeStreamingRange;
-@property (strong, nonatomic, nullable) VideoStreamingRange *supportedPortraitStreamingRange;
+@property (strong, nonatomic, nullable) SDLVideoStreamingRange *supportedLandscapeStreamingRange;
+@property (strong, nonatomic, nullable) SDLVideoStreamingRange *supportedPortraitStreamingRange;
+@property (weak, nonatomic, nullable) id<SDLStreamingMediaDelegate> delegate;
+```
+
+Where an object conforming to `SDLStreamingMediaDelegate` should implement the following methods:
+
+```objectivec
+@protocol SDLStreamingMediaDelegate <NSObject>
 
-@interface VideoStreamingRange : NSObject
+- (void)videoStreamingSizeDidUpdate:(CGSize)displaySize;
+- (void)videoStreamingSizeDoesNotMatch;
```

Comment:
```suggestion
```

This is not included in https://github.com/smartdevicelink/sdl_evolution/issues/983#issuecomment-642797352.  Please remove.